src/runtime/pickle_runner_spec.ts

```diff
@@ -302,33 +329,60 @@ describe('PickleRunner', () => {
               attempt: 0,
               id: '2',
               testCaseId: '0',
+              timestamp: {
+                nanos: 1000000,
+                seconds: {
+                  high: 0,
+                  low: 0,
+                  unsigned: false,
+                },
+              },
```

Comment:
Probably worth creating a test helper `getTestTimestamp` that takes in the number of milliseconds to reduce the duplication